pandas/core/indexes/base.py

```diff
@@ -5456,28 +5456,61 @@ def _add_numeric_methods(cls):
         cls._add_numeric_methods_unary()
         cls._add_numeric_methods_binary()
 
-    @classmethod
-    def _add_logical_methods(cls):
-        """
-        Add in logical methods.
+    def any(self, *args, **kwargs):
         """
-        _doc = """
-        %(desc)s
+        Return whether any element is Truthy.
 
         Parameters
         ----------
         *args
-            These parameters will be passed to numpy.%(outname)s.
+            These parameters will be passed to numpy.any.
         **kwargs
-            These parameters will be passed to numpy.%(outname)s.
+            These parameters will be passed to numpy.any.
 
         Returns
         -------
-        %(outname)s : bool or array_like (if axis is specified)
-            A single element array_like may be converted to bool."""
+        any : bool or array_like (if axis is specified)
+            A single element array_like may be converted to bool.
 
-        _index_shared_docs["index_all"] = dedent(
-            """
+        See Also
+        --------
+        Index.all : Return whether all elements are True.
+        Series.all : Return whether all elements are True.
+
+        Notes
+        -----
+        Not a Number (NaN), positive infinity and negative infinity
+        evaluate to True because these are not equal to zero.
+
+        Examples
+        --------
+        >>> index = pd.Index([0, 1, 2])
+        >>> index.any()
+        True
+
+        >>> index = pd.Index([0, 0, 0])
+        >>> index.any()
+        False
+        """
+        # FIXME: docstr inaccurate, args/kwargs not passed
+        self._maybe_disable_logical_methods("any")
```

Comment:
do we have a numpy validation routine for this and for all? eg. https://github.com/pandas-dev/pandas/blob/master/pandas/compat/numpy/function.py

Reply:
no